UNRELEASED.md

```diff
@@ -10,6 +10,9 @@
          * Some bug fix, see #124
 
      DO NOT LEAVE A BLANK LINE BELOW THIS PREAMBLE -->
+### Design
```

Comment:
Not a blocker or anything, but I don’t think we need to include design documents in the release notes.

min my view, the audience for release notes are users who might experience changes or new features in the api. Anyone who cares about design and development rather than feature releases should get that info by subscribing to notifications about prs.